.ruby-version

```diff
@@ -1 +1,4 @@
+feature4/controllers
```

Comment:
* In the requirements you are required not to use rails generate scaffold command. The point of this project is to show you how to set up controllers by yourself.
* View files should include only placeholders written in plain HTML,
* Make sure that you added code to handle only the 4 required URLs, Kindly stick to all these requirements as currently, your code doesn't follow the requirements 
![image](https://user-images.githubusercontent.com/52497246/236996278-848b432a-29c4-4a99-b908-caa2bb97f6f5.png)
![image](https://user-images.githubusercontent.com/52497246/236996389-96105785-3d00-407a-97ad-e7efa35fed04.png)
![image](https://user-images.githubusercontent.com/52497246/236996409-87098e9c-de28-45be-a0db-abf5bd0c0af4.png)